### ISSUE TITLE
acme.sh dnsapi

### ISSUE THREAD

**Issue by mtzlobr** [Froxlor/Froxlor#1195]
please add the possibility to use dnsapi for acme.sh instead of Webroot mode

./acme.sh --issue --dns dns_myDNS -d example.com -d '*.example.com'

Thank you
mt

**Comment by d00p** [comment]
We could but it's not that easy because every dns-api-provider has it's own env-vars credentials / requirements and not every domain-registrar has an dns-api script within acme.sh

A pull request that fulfills all these mechanisms and requirements is welcomed

**Comment by mtzlobr** [comment]
i thought it would be a good idea to use acme.sh's internal functionality when there is a wide range of providers in the dnsapi directory.
in froxlor administration only the option to use dnsapi would be selected and the dns provider would be entered as a parameter.
handling credentials / dns-api-provider requests would be handled from acme.sh, that would probably mean manual configuration for acme.sh in the next step, but that's something that can be accepted
BR
mt

**Comment by d00p** [comment]
So tell me then how a customer should enter any credentials for the selected dns provider

**Comment by mtzlobr** [comment]
yes  it is a issue if it will be operate from customer site, you have true.
my thought was manually from the console on the server.
I assumed it was an admin issue. somehow it didn't occur to me that the customer has his own dns provider and might want to set it up himself.
but probably even that could be solved by creating a copy of the given dnsapi in the client's home where it could be configured and lik it beck to acme.sh.
but it is my spekulation how to do it now.

**Comment by ardyvictoire** [comment]
comment ajouter une client(dns) dans le froxlor
